docs/Meadow/Getting_Started/Deploying_Meadow/index.md

```diff
@@ -39,14 +39,26 @@ From a console with admin rights, execute following command:
 meadow install dfu-util
 ```
 
-### Update USB driver for ST devices
+### Ensure Meadow drivers are correct before deploying Meadow.OS
```

Comment:
Maybe this should be 

### Install Meadow USB drivers 

Short and specific ... the old title is in the weeds, the new title is trying to do a little too much work imo

Reply:
I like your title suggestion better, thanks.